user-impl/src/main/java/com/example/auction/user/impl/AuthEvent.java

```diff
@@ -0,0 +1,24 @@
+package com.example.auction.user.impl;
+import com.example.auction.user.api.Auth;
+import com.example.auction.user.api.User;
+import com.lightbend.lagom.javadsl.persistence.AggregateEvent;
```

Comment:
As a general rule, avoid reusing API classes in PersistentEntity-related operations. This is a best practice knonw as Anti-corruption layer.

In authentication in particular you will find that almost every message exchanged to-from the service differs. For example, creating credentials may need two passwords, a username and some other data but logging in only needs one password and a username. Then, in no case should the service send a password in a response...
... you get the gist ;-)